netebpfext/net_ebpf_ext_tracelog.h

```diff
@@ -209,16 +209,90 @@ net_ebpf_ext_trace_terminate();
         TraceLoggingUInt64((value2), (#value2)),                                                             \
         TraceLoggingUInt64((value3), (#value3)));
 
-#define NET_EBPF_EXT_LOG_SOCK_ADDR_CLASSIFY(trace_level, keyword, message, handle, protocol, redirect, verdict) \
-    TraceLoggingWrite(                                                                                          \
-        net_ebpf_ext_tracelog_provider,                                                                         \
-        NET_EBPF_EXT_TRACELOG_EVENT_GENERIC_MESSAGE,                                                            \
-        TraceLoggingLevel(trace_level),                                                                         \
-        TraceLoggingKeyword((keyword)),                                                                         \
-        TraceLoggingString(message, "Message"),                                                                 \
-        TraceLoggingUInt64((handle), "TransportEndpointHandle"),                                                \
-        TraceLoggingUInt64((protocol), "Protocol"),                                                             \
-        TraceLoggingUInt64((redirect), "Redirected"),                                                           \
+#define NET_EBPF_EXT_LOG_SOCK_ADDR_CLASSIFY_IPV4(                                                              \
```

Comment:
move these to `net_ebpf_ext_sock_addr.h`.

Reply:
Shouldn't all tracing macros be in one file, even if some of them are sock_addr specfic?

Reply:
Given that these macros are not dependent on specific header files, I'd personally prefer all tracing macros to be in one file as well.

Reply:
Disagree. Module specific tracing should be local to the module. I will fix this.